src/test/java/de/tum/in/www1/artemis/programmingexercise/ProgrammingExerciseResultBambooIntegrationTest.java

```diff
@@ -24,8 +28,6 @@
 
     @BeforeEach
     void setup() {
-        programmingExerciseResultTestService.setup();
```

Comment:
please revert this change and keep the setup method here!

Reply:
done in ba1458f587a5d5b3c9c465d0ab19e1c675d5d3f0